lang/r/r.talon

```diff
@@ -52,19 +52,19 @@ action(user.code_import):
     key(left)
 action(user.code_comment): "#"
 action(user.code_state_return):
-	 insert("return()")
+   insert("return()")
    key(left)
 action(user.code_break): "break"
 action(user.code_next): "next"
 action(user.code_true): "TRUE"
 action(user.code_false): "FALSE"
 
 # R specific commands
-chain:
+(chain|pipe that):
     key(end)
     " %>%"
     key(enter)
 state na:
     insert("NA")
 
-^funky <user.text>$: user.code_private_function(text)
+^function define <user.text>$: user.code_private_function(text)
```

Comment:
I agree this is better than existing since funky is already defined in `programming.talon`. As a separate issue, we should probably move this line into `programming.talon` as well. No need for that to gate this PR though.